src/snowflake/connector/ocsp_snowflake.py

```diff
@@ -364,7 +365,8 @@ def download_cache_from_server(self, ocsp):
                 # len(OCSP_RESPONSE_VALIDATION_CACHE) is thread-safe, however, we do not want to
                 # block for logging purpose, thus using len(OCSP_RESPONSE_VALIDATION_CACHE._cache) here.
                 logger.debug(
-                    "# of certificates: %s", len(OCSP_RESPONSE_VALIDATION_CACHE._cache)
+                    "# of certificates: %s",
```

Comment:
`%s` is for strings, `%i`, or `%u` might be more appropiate
```suggestion
                    "# of certificates: %u",
```

Reply:
updated